PocketKit/Sources/PocketKit/Notifications/PocketBraze.swift

```diff
@@ -56,19 +58,31 @@ class PocketBraze: NSObject {
  Conforming to our PocketBraze Protocol
  */
 extension PocketBraze: BrazeProtocol {
+    func isFeatureFlagEnabled(id: String) -> Bool {
+        guard let featureFlag = braze.featureFlags.featureFlag(id: id) else {
+            return false
+        }
+
+        return featureFlag.enabled
+    }
+
     func loggedIn(session: SharedPocketKit.Session) {
         DispatchQueue.main.async { [weak self] in
+            guard let self else { return }
             // Braze SDK docs say this needs to be called from the main thread.
             // https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/analytics/setting_user_ids/#assigning-a-user-id
-            self?.braze.changeUser(userId: session.userIdentifier)
+            braze.changeUser(userId: session.userIdentifier)
             // Was this build deployed through the App Store or through TestFlight?
             var isTestFlight = false
             if let receiptURL = Bundle.main.appStoreReceiptURL {
                 isTestFlight = receiptURL.path(percentEncoded: false).contains("sandboxreceipt")
             }
             // Could expand to include "development"
             let deployment = isTestFlight ? "testflight" : "app_store"
-            self?.braze.user.setCustomAttribute(key: "ios_deployment", value: deployment)
+            braze.user.setCustomAttribute(key: "ios_deployment", value: deployment)
```

Comment:
btw @nzeltzer i dont think this works, ive noticed you and I are set to appstore in braze :(